WorkloadManagementSystem/PilotAgent/pilotTools.py

```diff
@@ -288,7 +333,13 @@ def __init__( self, pilotParams, dummy='' ):
     """
 
     self.pp = pilotParams
-    self.log = Logger( self.__class__.__name__ )
+    self.log = ExtendedLogger(
```

Comment:
When calling a method/function with named attributes, it is good practice to name the attributes as well